package.json

```diff
@@ -35,6 +36,18 @@
     "karma-phantomjs-launcher": "^0.2.0",
     "karma-requirejs": "^0.2.2",
     "phantomjs": "^1.9.17",
+    "remark-cli": "^3.0.0",
+    "remark-lint": "^6.0.0",
+    "remark-preset-lint-consistent": "^2.0.0",
+    "remark-preset-lint-recommended": "^2.0.0",
+    "remark-validate-links": "^6.0.0",
     "requirejs": "^2.1.18"
+  },
+  "remarkConfig": {
+    "plugins": [
+      "preset-lint-recommended",
```

Comment:
Default value for list-item-indent rule is creating a lot of irrelevant problems. Consider adding the following to the "plugins" array:

```json
[
    "remark-lint-list-item-indent",
    "mixed"
],
```